docs/solutions/ha-setup-apt.md

```diff
@@ -404,14 +482,14 @@ postgres=#
 
 ## Configure HAProxy
 
-HAProxy node will accept client connection requests and route those to the active node of the PostgreSQL cluster. This way, a client application doesn’t have to know what node in the underlying cluster is the current primary. All it needs to do is to access a single HAProxy URL and send its read/write requests there. Behind-the-scene, HAProxy routes the connection to a healthy node (as long as there is at least one healthy node available) and ensures that client application requests are never rejected. 
+HAProxy node accepts client connection requests and route those to the active node of the PostgreSQL cluster. This way, a client application doesn’t have to know what node in the underlying cluster is the current primary. All it needs to do is to access a single HAProxy URL and send its read/write requests there. Behind-the-scene, HAProxy routes the connection to a healthy node (as long as there is at least one healthy node available) and ensures that client application requests are never rejected. 
 
 HAProxy is capable of routing write requests to the primary node and read requests - to the secondaries in a round-robin fashion so that no secondary instance is unnecessarily loaded. To make this happen, provide different ports in the HAProxy configuration file. In this deployment, writes are routed to port 5000 and reads  - to port 5001.
```

Comment:
This paragraph is a bit confusing. If one reads it without proper attention will think HAProxy is able to split and distribute the writes and reads on all nodes. I think we need to improve to make sure it is clear the HAProxy will send all the writes to the primary node and the reads can be distributed among all nodes. The secret is that we use 2 ports, one for the primary, that is able to accept reads and writes and all connections will be routed to the primary and another port used to distribute read-only queries that go to the replicas. This second port HAProxy will balance the load among all replicas and the default method is using round-robin, but there are other implementations.

Reply:
Amended